conference/react-js/src/components/UpdateTalk/update-talk.scss

```diff
@@ -7,24 +7,30 @@
 		box-shadow: 0 1px 5px 0 rgba(0, 0, 0, .15);
 	}
 
-	&__title {
-		display: flex;
-		align-items: center;
-
-		&-item {
-			flex-grow: 1;
-		}
-	}
-
 	&__button {
-		margin-left: 12px;
+		//display: flex;
+		//align-items: center;
+		//
+		//&-item {
+			margin-right: 25px;
+		//}
 	}
 
 }
 
 .select-field {
-	&_wrapper {
-		display: flex;
-		justify-content: space-between;
+
+	&__item {
+		margin-right: 20px;
+	}
+
+	&__topic {
+		width: 350px !important;
```

Comment:
Can we avoid "!important" here?

Reply:
Unfortunately we can`t  because material-ui styles interrupt this style rules